config/foreman-proxy-content.migrations/251105202301-reset-puppet-package-params.rb

```diff
@@ -0,0 +1,15 @@
+if answers["puppet"].is_a?(Hash)
+  puppet = answers["puppet"]
+
+  if puppet.has_key?("server_package") &&
+     (puppet["server_package"].nil? || puppet["server_package"] == "" ||
```

Comment:
empty? works for strings too, so you could just do `.nil? || .empty?`

OTOH, setting it to nil when it already is nil seems odd, but then you have to check that it supports empty (or is not nil 🙃), so the above is probably still the nicest shortcut